.github/workflows/generator_container_slsa3.yml

```diff
@@ -48,12 +52,17 @@ on:
         required: false
         type: boolean
         default: false
```

Comment:
I think for #1177 we want to add our own `continue-on-error` input to the workflow that defaults to `false` (for backwards compatibility). This gives the user more control over what they want to do and doesn't force them to check `outcome` if they want to fail fast.

This input would be passed to the `continue-on-error` inputs further down. `outcome` would thus only be valid if the user passed `continue-on-error: true` to the reusable workflow.

Reply:
OK. So `continue-on-error` cannot take a variable as input. So I'll set them all to `true` and add a final step in each job that will exit (or not) based on the value of `continue-on-error`. Correct?

Reply:
Lovely. Yeah, I suppose that will have to be how it's implemented...

Reply:
PTAL. I still needed the additional `final` job with `always()`, otherwise the overall run would be reported as a failure.

Reply:
BTW, the docs seem to imply that `inputs` can be used in `jobs.<job_id>.steps.continue-on-error`. I wonder if the docs are wrong or is it just not available for reusable workflows? I assume you tried it and got an error?
https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability

Reply:
You can pass it, but it's interpreted as a string rather than a boolean and the workflow is deemed invalid. I tried `${{ inputs.continue-on-error == 'true' }}` but still was complaining about not being a boolean. Let me know if you have other ideas.